app/controllers/users.js

```diff
@@ -269,3 +271,69 @@ exports.user = (req, res, next, id) => {
       next();
     });
 };
+
+/**
+ * Invite user to play game.
+ */
+const sgMail = require('@sendgrid/mail');
```

Comment:
I think the variable naming convention here is a little off can use `sendGridMailer` instead. @Rhotimee